app/domain/people_skills_filter.rb

```diff
@@ -1,15 +1,20 @@
 # frozen_string_literal: true
 
 class PeopleSkillsFilter
-  attr_reader :entries, :rated
+  attr_reader :entries, :rated, :level
 
-  def initialize(entries, rated)
+  def initialize(entries, rated, level = nil)
     @entries = entries
     @rated = rated
+    @level = level
   end
 
   def scope
-    filter_by_rated
+    if level.nil?
+      filter_by_rated
+    else
+      filter_by_level
+    end
```

Comment:
I wouldn't do the conditional here. Just call both methods. But you need to pass the entries you received from the `filter_by_rated` method to the `filter_by_level` method. Thus, change both method declarations to having the entries as parameters and then work with the return value.